kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -855,7 +876,7 @@ class ResidualBasedBlockBuilderAndSolver
      */
     void Clear() override
     {
-       BaseType::Clear();
+        BaseType::Clear();
 
         mSlaveIds.clear();
         mMasterIds.clear();

```

Comment:
you should also clear mInactiveSlaveDofs

Reply:
True